teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md

```diff
@@ -22,23 +22,16 @@ Assigns Teams Media Logging policy to a user, group of users or the entire tenan
 
 ```
 Grant-CsTeamsMediaLoggingPolicy [[-Identity] <String>] [-PassThru] [[-PolicyName] <String>]
- [-MsftInternalProcessingMode <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+[-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### GrantToTenant
 
 ```
-Grant-CsTeamsMediaLoggingPolicy [-PassThru] [[-PolicyName] <String>] [-MsftInternalProcessingMode <String>]
+Grant-CsTeamsMediaLoggingPolicy [-PassThru] [[-PolicyName] <String>]
  [-Global] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-### GrantToGroup
```

Comment:
It seems that Group and Rank parameters are available. Why remove them?

Reply:
I checked `-Group` and `-Rank` parameters with this cmdlet as well as some other Grant-<..> cmdlets, and it seems that those are not implemented as I'm getting 'A parameter cannot be found' error.